workflows/chipseq/Snakefile

```diff
@@ -684,7 +726,8 @@ rule bed_to_bigbed:
 
 
         # bedToBigBed doesn't handle zero-size files
-        if os.stat(input.bed).st_size == 0:
+        # bigbed is not created from epic2-generated peaks
+        if os.stat(input.bed).st_size == 0 or "epic2" in input.bed:
```

Comment:
Not clear why we don't get a bigbed from epic2's bed. Does the bed file it creates not work with this rule? If that's the case we should fix it, possibly in a wrapper if needed (if the rule gets too messy/complicated).

Reply:
I think the `bedToBigBed` cannot handle the format of epic2-created `bed` files.

* macs2

```bash
chr2L   5208    6064    gaf-wingdisc-pooled_peak_1      198     .       10.3022 22.4865 19.8284 464
chr2L   7465    7612    gaf-wingdisc-pooled_peak_2      23      .       3.56078 4.31716 2.38755 67
```

* epic2 

```bash
chr2L   5000    12399   1.59536939537213e-31    181.4181186791179       .       123     37      3.464230687093768e-301.814181186791179
chr2L   65000   83599   1.0311678411768225e-52  146.05718430892273      .       307     118     3.13475023717754e-51 1.4605718430892272
```

One way of resolving it is to convert epic2-bed to `bedToBigBed` compatible format using the `pybedtools`.